Cooper/ORM/ICRUD.cs

```diff
@@ -8,11 +8,13 @@ namespace Cooper.ORM
     internal interface ICRUD
     {
         
-        long Create(string table, string idColumn, EntityORM attributeValue); //return column id or 0 for verification
+        long Create(string table, string idColumn, EntityORM entity, bool returning = true); //return column id or 0 for verification
 
-        EntityORM Read(object attribute_value, string attribute_name, HashSet<string> attributes, string table);
+        EntityORM Read(object attribute_value, string attribute_name, HashSet<string> attributes, string table); //Read unique field
 
-        IEnumerable<EntityORM> ReadAll(string table, HashSet<string> attributes);
+        IEnumerable<EntityORM> ReadAll(object attribute_value, string attribute_name, HashSet<string> attributes, string table); //Read entities by parameter
```

Comment:
Will we ever have multiple filter criteria? If so, may be we should have arrays for attribute_value and _name

Reply:
Yeah, but we can add them, while it will need.